amt/site/templates/parts/filter_list.html.j2

```diff
@@ -0,0 +1,46 @@
+<div class="margin-top-large">
+    <span>3 {% trans %}results for{% endtrans %}</span>
+    <form name="htmx-remove-tags"
```

Comment:
I was hoping we could use the same form for this?

Reply:
Maybe.. it became a bit cumbersome because the initial form adds the filters so to also remove them makes it mingle unwantedly.. besides that, this way we can update the filters above according to the removal..